maintenance_plan/views/maintenance.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <!-- maintenance.kind : views -->
+    <record id="maintenance_kind_view_tree" model="ir.ui.view">
+        <field name="name">maintenance.kind.tree</field>
+        <field name="model">maintenance.kind</field>
+        <field name="arch" type="xml">
+            <tree editable="top">
+                <field name="name"/>
+                <field name="active"/>
+            </tree>
+        </field>
+    </record>
+
+    <!-- maintenance.kind : actions -->
+    <record id="maintenance_kind_action" model="ir.actions.act_window">
+        <field name="name">Maintenance Kinds</field>
+        <field name="res_model">maintenance.kind</field>
+        <field name="view_mode">tree</field>
+        <field name="view_id" ref="maintenance_kind_view_tree"/>
+    </record>
+
+    <menuitem
+        id="menu_maintenance_kind_configuration"
+        parent="maintenance.menu_maintenance_configuration"
+        action="maintenance_kind_action"
+        sequence="4" />
+
+
+    <!-- maintenance.plan : views -->
+    <record id="maintenance_plan_view_form" model="ir.ui.view">
+        <field name="name">maintenance.plan.form</field>
+        <field name="model">maintenance.plan</field>
+        <field name="arch" type="xml">
+            <form>
+                <group>
+                    <field name="maintenance_kind_id"/>
+                    <field name="period"/>
+                    <field name="duration"/>
+                    <field name="next_maintenance_date"/>
+                </group>
+            </form>
+        </field>
+    </record>
+
+    <record id="maintenance_plan_view_tree" model="ir.ui.view">
+        <field name="name">maintenance.plan.tree</field>
+        <field name="model">maintenance.plan</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="maintenance_kind_id"/>
+                <field name="period"/>
+                <field name="duration"/>
+                <field name="next_maintenance_date"/>
+            </tree>
+        </field>
+    </record>
+
+    <!-- equipment : views inheritance -->
+    <record id="hr_equipment_view_form" model="ir.ui.view">
+        <field name="inherit_id" ref="maintenance.hr_equipment_view_form" />
+        <field name="model">maintenance.equipment</field>
+        <field name="arch" type="xml">
+            <xpath expr="//group[@name='maintenance']" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
+            <xpath expr="//group[@name='maintenance']" position="after">
+                <group name="maintenance">
```

Comment:
You should give this new group a different name, because the old "maintenance" group is now invisible and so it's still present.

Reply:
Ok, fixed